### ISSUE TITLE
Warning message and incorrect plot for plotTemporalSymmetry when timescale = days is set in summariseTemporalSymmetry

### ISSUE THREAD

**Issue by daniellenewby** [oxford-pharmacoepi/CohortSymmetry#261]
I want to create a plot of the temporal sequence of my intersection of my index and markers. 
I can create the correct plot using the following code:

```
temporal_symmetry <- summariseTemporalSymmetry(cohort = cdm$ache_inhibitors_memantine,
                                               timescale = "month")

plotTemporalSymmetry(result = temporal_symmetry,
                     xlim = c(-6, 6))

```

This creates the following plot:
![image](https://github.com/user-attachments/assets/76c2186f-9f85-4221-a5d3-eeab2981656f)

This is what i would expect. However if i put the following code to have this in days:

```
temporal_symmetry <- summariseTemporalSymmetry(cohort = cdm$ache_inhibitors_memantine,
                                               timescale = "day")


plotTemporalSymmetry(result = temporal_symmetry,
                     labs = c("Time (days)", "Individuals (N)"),
                     xlim = c(-182, 182))
```

I get the following plot which is incorrect as its missing information on the LHS.

![image](https://github.com/user-attachments/assets/0c6a7856-23fe-4cd5-bede-49f2f76d95ee)

In the R studio console i get the following warning message:
"Warning messages:
1: `position_stack()` requires non-overlapping x intervals. 
2: Removed 144 rows containing missing values or values outside the scale range (`geom_col()`). 
3: Removed 144 rows containing missing values or values outside the scale range (`geom_point()`)."

Ive checked the outputs from temporal_symmetry and these seem correct (i.e i have values from -182 to 182. Therefore i think this is a bug with the plotting function.

**Comment by daniellenewby** [comment]
This is using the CRAN version of the package

**Comment by daniellenewby** [comment]
@KimLopezGuell think this is your expertise :)

**Comment by KimLopezGuell** [comment]
Hi Danielle! I do not know what data you used so I cannot reproduce your specific example. Before I dive more into this, after taking a look in my local, I wonder whether the issue might be minCellCount in summariseTemporalSymmetry()? Have you tried setting it to 0? Maybe you get many counts <5 which are then turned into NA and that is why you do not see anything in the plot.

**Comment by daniellenewby** [comment]
yep that works! Thank you :) @xihang-chen can we mention this in the vignette stating that min cell count in the function can impact the plot? or not have the min cell count in the function and users can suppress the counts afterwards running this function?

**Comment by KimLopezGuell** [comment]
I think the idea would be to deprecate minCellCount and do all the suppressing when exporting the results (i.e in omopgenerics). Not sure about the plot function though. @catalamarti ?
